### PR TITLE
Add php connector

### PR DESCRIPTION
**Pull request by Yurunsoft**

### CODE REVIEW

**Comment by sangshuduo** [comment]
Thanks for your contribution. Would you like to add your connector as a submodule of TDengine?

**Comment by Yurunsoft** [comment]
No problem.
If necessary, I can also transfer the project to taosdata.

**Comment by sangshuduo** [comment]
> No problem. If necessary, I can also transfer the project to taosdata.

I notice you use the Apache license. I don't know if the PHP team would encourage the connector to use GPL as you might know the TDengine use AGPL。

**Comment by sangshuduo** [comment]
BTW, would you like to add a few test cases into `TDengine/tests` to make the CI check if PHP is not broken after someone submit new code?

**Comment by Yurunsoft** [comment]
OK, I'll change license later and add tests

I'll close this PR first and reopen it after it's finished

**Comment by Yurunsoft** [comment]
I have changed PHP tdengine license to AGPLv3.

I don't know how to add test cases and debugging. The Ci of tdengine doesn't seem to run in GitHub actions

**Comment by sangshuduo** [comment]
> I have changed php-tdengine license to AGPLv3.
> 
> I don't know how to add test cases and debugging. The CI of tdengine doesn't seem to run in GitHub actions

Please add your test case and document how to run it. We will take care of them to add to our CI.

File: .gitmodules [hunk 1]
```diff
@@ -20,3 +20,9 @@
 [submodule "examples/rust"]
 	path = examples/rust
 	url = https://github.com/songtianyi/tdengine-rust-bindings.git
+[submodule "src/connector/php-tdengine"]
+	path = src/connector/php-tdengine
+	url = https://github.com/Yurunsoft/php-tdengine.git
+[submodule "src/connector/tdengine-restful-connector"]
```
Comment (sangshuduo) [inline .gitmodules]: I think tdengine-restful-connector is a general name that might confuse the user. I suggest using a name with 'php' to be more indicative like another one you submitted.

**Review by sangshuduo — CHANGES_REQUESTED** [review]
a few comments

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**